docs/glossary.md

```diff
@@ -54,10 +58,18 @@ title: Glossary
 
 **Proof-of-Mixing** - The reward of NYM tokens is based on the good quality of service carried out by node operators.
 
+**Public Key** - A large numerical value that is used to encrypt data.
```

Comment:
If we have an entry for public key, then shouldn't we also have one for `private key`? :)